recipes/taxidTools/meta.yaml

```diff
@@ -0,0 +1,45 @@
+{% set version = "2.2.3" %}
+
+package:
+  name: taxidtools
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/t/taxidTools/taxidTools-{{ version }}.tar.gz
+  sha256: c1add40e166eaa47fc0802ab3f5b4af64a8b03560151d527a5c35eba15d105ae
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  build:
```

Comment:
It does not appear as if this package is building any C-extensions or compiling code, so the `build` section under `requirements` can be removed.

Reply:
Thanks for the advice, I removed the build section under requirements altogether as no dependencies is required to build the package.